devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy

```diff
@@ -23,6 +23,7 @@ metadata {
         capability "Health Check"
         capability "Sensor"
         capability "Configuration"
+        capability "Power Consumption Report"

```

Comment:
1. If this capability has to be exposed please create a separate DTH that will handle it (and move the fingerprints that will use it)
zigbee-metering-plug-power-consumption-report.src/zigbee-metering-plug-power-consumption-report.groovy 

2. Please direct this pull request to the "staging" branch

Reply:
I changed the branch after pressing the 'New full request' button (compare: master -> staging).

* Commits on Nov 04, 2020
- Merge pull request #49103 from SmartThingsCommunity/master
- Rolling up master to staging

When I pressed the 'Create full request' button, I got a message below.
"Pull request creation failed. Validation failed: You can't performance that action at this time."

Reply:
1. I changed the branch after pressing the 'New full request' button (compare: master -> staging).

* Commits on Nov 04, 2020
- Merge pull request #49103 from SmartThingsCommunity/master
- Rolling up master to staging

When I pressed the 'Create full request' button, I got a message below.
"Pull request creation failed. Validation failed: You can't performance that action at this time."

2. I created 'Zigbee Metering Plug Power Consumption Report' DTH in IDE.
And I changed DTH's Repository to 'staging'.
I made a commit change but got the result: "404 ERROR: Not Found."

Reply:
ok, so please (for now) push Your changes to this (master) branch.

Reply:
I commited to my master branch.